Add webview_android browser data

Elchi3 opened the pull request:
The fixes the last item on #591: Add release data (allowed versions) for `webview_android` and validate our compat data with it.

Some notes here:
- Android 4.4 is based on Chrome 30.
- Android 4.4.3 is based on Chrome 33.
- Android 5 had a webview based on Chrome 37 but also decoupled OS and browser, so the web engine can be updated separately. That means we can use Chrome Android version numbers from that point on.

@jpmedley can you have a look here? Does this allow-list for `webview_android` versions make sense to you?

jpmedley commented:
No because you've only listed the version of Chrome that Android shipped with. After the phone is activated, Chrome and webview auto-update with every Chrome release.

Elchi3 commented:
> No because you've only listed the version of Chrome that Android shipped with. After the phone is activated, Chrome and webview auto-update with every Chrome release.

Not sure I understand how this is contrary to what this PR does.

This PR adds valid WebView versions. WebView used to be part of the Android OS and so we're referring to the Android version for that period (caniuse does this as well btw). Some WebViews were already based on Chrome. I know of two here. Still we're going with the Android versions, as these are more commonly referred to when the WebView was not decoupled from Android OS. So, that leaves us with these:
- 2.2
- 2.3
- 3
- 4
- 4.1
- 4.2
- 4.4 ( WebView based on Chrome 30)
- 4.4.3 (WebView based on Chrome 33)

Then in Android 5, WebView was decoupled from Android OS and now auto updates with every Chrome release. So, instead of adding Android 5 as a valid version, we refer to the first version of Chrome that was available on Android 5 (this is 37). And now that we're decoupled from Android, we can start using Chrome versions until the current one. So, we end up with all of these:
- 37
- 38
- 39
- ...
- 69

Please see the diff of this PR for all versions added.

jpmedley commented:
Sorry, I have a tendency to go on autopilot regarding the webview issue. It's the result of too many discussions with people who think webview is still tied to Android.

Elchi3 commented:
There are also a few early android versions that already included a web browser. So, maybe we should add these as well. https://en.wikipedia.org/wiki/Android_version_history#Version_history_by_API_level
- 1
- 1.1
- 1.5
- 1.6

"1" would then be a valid first version which might make sense in a lot of cases. See #2717

Opinions?

Elchi3 commented:
@wbamberg, this is similar to https://github.com/mdn/browser-compat-data/pull/1356. See above on how I propose to go with versions here (we're following more or less caniuse). I would appreciate a review, but let me know if I should find someone else. Thank you!

Elchi3 commented:
Nice catches, @wbamberg ! :+1: 
The mapping you've identified is what I've tried to follow here.

Updated the instances that didn't follow these rules and removed the non-existent release notes.

wbamberg commented:
Your changes look good to me! There were some merge conflicts in Event.json, which I've tried to fix. But now there's also a linting error, where [`Event.stopImmediatePropagation` has "version_added": "6" for webview_android ](https://github.com/mdn/browser-compat-data/blob/c73ca86c6d10e3bb847c93c49e22baeb0420932a/api/Event.json#L1129).

Elchi3 commented:
I've rebased again. I think we're ready to merge now.
